src/commands/basemaps-github/__test__/create-manifest.test.ts

```diff
@@ -0,0 +1,21 @@
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
+
+import prettier from 'prettier';
+
+import { DefaultPrettierFormat } from '../make.cog.github.js';
+
+describe('DefaultPrettierFormat', () => {
+  it('should be the same prettier format with @linzjs/style', async () => {
+    const cfg = await prettier.resolveConfigFile('@linzjs/style/.prettierrc.cjs');
+    if (cfg == null) assert.ok(cfg !== null, 'Failed to read @linzjs/style/.prettierrc.cjs');
```

Comment:
```suggestion
    assert.ok(cfg, 'Failed to read @linzjs/style/.prettierrc.cjs');
```

Reply:
Nice, I don't know that asser.ok can automatically check non exist. I just fixed them.